test/integration/hook-err.spec.js

```diff
@@ -18,7 +19,72 @@ describe('hook error handling', function() {
   describe('before hook error tip', function() {
     before(run('hooks/before-hook-error-tip.fixture.js', onlyErrorTitle()));
     it('should verify results', function() {
-      expect(lines, 'to equal', ['1) spec 2', '"before all" hook:']);
+      expect(lines, 'to equal', [
+        '1) spec 2',
+        '"before all" hook for "skipped":'
+      ]);
+    });
+  });
+
+  describe('before hook root error', function() {
+    it('should verify results', function(done) {
+      runJSON('hooks/before-hook-root-error.fixture.js', [], function(
+        err,
```

Comment:
Suggest minor change to avoid ^%*^&% prettier reformatting...
```js
      var fixture = 'hooks/before-hook-root-error.fixture.js';
      runJSON(fixture, [], function(err, res) {
```

Reply:
I inserted `// eslint-disable-next-line`

Reply:
Please remove the `eslint` directive -- we use them for enforcement, not formatting